models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -72,7 +72,7 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
     # get the run dates based on the sipnet output.  we assume that even if the run is partial, the origin is still day 1 of the subset year
     start_month <- lubridate::month(as.Date(sub.sipnet.output$day[1], origin = paste0(y,"-01-01"))) # gets month
     ## using this approach to attempt to deal with inconsistent start index 0/1
-    model_start_date <- base::as.Date(paste0(y,"-",start_month,"-",ifelse(sub.sipnet.output$day[1]==0,1,sub.sipnet.output$day[1])))  
+    model_start_date <- base::as.Date(ifelse(sub.sipnet.output$day[1]==0,1,sub.sipnet.output$day[1]), origin = paste0(y,"-01-01"))
```

Comment:
doesn't this assume that we always start on January. What if I run a model from June to September?

Reply:
> No, I was running it with a model for November and it returned the correct date. The Jan 1 origin is so that as.Date can convert a day of year to the proper date. So I was using DOY= 317 and the start date returned was 2018-11-14.